docs/io/unified.rst

```diff
@@ -318,6 +327,78 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
+by Astropy by following the rules and conventions set by the standard. Thus,
+time coordinate columns in arbitrary FITS files (compliant with the standard)
+will be read as native `~astropy.time.Time` objects.
+
+However, a major chunk of the astronomical datasets, most of which are dated
+prior to the FITS time standard, follow different conventions to store time
+coordinate columns.
+
+A common convention found in existing FITS files is that a FITS binary
+table column with ``TTYPEn = ‘TIME’`` represents a time coordinate column.
+The FITS time standard states that such a column will be controlled by
+the global time reference frame keywords, and this will still be compliant
+with the present standard.
+
+Using this convention, Astropy can read time coordinate columns from all
+such FITS tables as native `~astropy.time.Time` objects. Common examples of
+FITS files following this convention are Chandra, XMM, and HST files.
+
+The following is an example of a Header extract of a Chandra event list:
+
+.. parsed-literal::
+
+    COMMENT      ---------- Globally valid key words ----------------
+    DATE    = '2016-01-27T12:34:24' / Date and time of file creation
+    TIMESYS = 'TT      '           / Time system                                    
+    MJDREF  =  5.0814000000000E+04 / [d] MJD zero point for times
+    TIMEUNIT= 's       '           / Time unit
+    TIMEREF = 'LOCAL   '           / Time reference (barycenter/local)
+
+    COMMENT      ---------- Time Column -----------------------
+    TTYPE1  = 'time    '           / S/C TT corresponding to mid-exposure           
+    TFORM1  = '1D      '           / format of field                                
+    TUNIT1  = 's       '
+
+When reading such a FITS table with ``astropy_native=True``, Astropy checks
+whether the name of a column is "TIME"/ "time" (``TTYPEn = ‘TIME’``) and
+whether its unit is a FITS recognized time unit (``TUNITn`` is a time unit).
+
+For example, reading a Chandra event list which has the above mentioned header
+and the time coordinate column ``time`` as ``[1, 2]`` will give::
+
+    >>> from astropy.time import Time, TimeDelta
+    >>> from astropy.table import Table
+    >>> tm = Table.read('chandra_events.fits', astropy_native=True)
+    WARNING: Time column "time" reference position will be ignored due to unspecified observatory position. [astropy.io.fits.fitstime]
+    >>> tm['time']
+    <Time object: scale='tt' format='mjd' value=[ 50814.00001157  50814.00002315]>
+    >>> # MJDREF  =  5.0814000000000E+04, TIMESYS = 'TT'
+    >>> ref_time = Time(5.0814000000000E+04, format='mjd', scale='tt')
+    >>> # TTYPE1  = 'time', TUNIT1 = 's'
+    >>> delta_time = TimeDelta([1, 2], format='sec')
+    >>> ref_time + delta_time == tm['time']
+    array([ True,  True], dtype=bool)
+
+By default, FITS table columns will be read as standard `~astropy.table.Column`
+objects without taking the FITS time standard into consideration.
+
```

Comment:
Add a subsection header `String column in ISO-8601 format` here, then describe it.  Include description of the criteria for identifying the column as Time.

Are these two conventions the only ones?